.gitignore

```diff
@@ -34,4 +34,7 @@ staticfiles/
 .coverage
 
 # Qahtan's Notes
-qahtansnotes.txt
\ No newline at end of file
+qahtansnotes.txt
```

Comment:
Can we remove member specific files from this to keep the code clean?

Reply:
Sure, I had it in the .gitignore but can move elsewhere!